src/Symfony/Component/Messenger/Adapter/PhpAmqp/Exception/RejectMessageException.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Adapter\PhpAmqp\Exception;
+
+/**
+ * If something goes wrong while consuming and handling a message from the AMQP broker, there are two choices: rejecting
+ * or re-queuing the message.
+ *
+ * If the exception that is thrown by the bus while dispatching the message implements this interface, the message will
+ * be rejected. Otherwise, it will be re-queued.
+ *
+ * @author Samuel Roze <samuel.roze@gmail.com>
+ */
+interface RejectMessageException extends \Throwable
```

Comment:
Throwable cannot be extended https://3v4l.org/E7eQ6 , also it misses the interface suffix

Reply:
That's the point: this interface should only be implemented by classes that are extending `\Exception`: https://3v4l.org/belYf.

Agreed with the `Interface` suffix though.

Reply:
Right!